src/components/ProfileBox/ProfileBox.module.scss

```diff
@@ -49,9 +49,8 @@
         justify-content: center;
         align-items: center;        
         // border: 1px solid rgba(255, 255, 255, 0.3);
-        background: #1F232C;
-        border: 1px solid transparent;
-        border-radius: 100px;
+        border: 1px solid white;
+        border-radius: 75px;
```

Comment:
You could create an SCSS variable (on top of this file) for the avatar size used for multiple properties.

```scss
$avatar-size: 75px;

...
```

Reply:
db138a252ab26d75a6f649041fff24356614ca05